script.js

```diff
@@ -0,0 +1,150 @@
+const toDoForm = document.querySelector(".toDoForm");
+const toDoInput = toDoForm.querySelector("input");
+const toDoList = document.querySelector(".toDoList");
+const finishedList = document.querySelector(".finishedList");
+const lenTodo = document.getElementById("todo-count");
+const lenFin = document.getElementById("fin-count");
+
+const TODODATA = "todoData"
+const FINDATA = "FinData"
+
+let toDoArr = [];
+let finArr = [];
+
+function loadDATA(){
+    const loadedTODO = localStorage.getItem(TODODATA);
+    const loadedFIN = localStorage.getItem(FINDATA);
+    
+    if (loadedTODO !== null) {
+        const parsedTODO = JSON.parse(loadedTODO); // parse, 가져온 것을 자바스크립트 object로 변환시켜줌
+        parsedTODO.forEach(function(input) {
+            printData(input.text,parseInt(input.id),true)
+        });
+    }
+    if (loadedFIN !== null) {
+        const parsedFIN = JSON.parse(loadedFIN); // parse, 가져온 것을 자바스크립트 object로 변환시켜줌
+        parsedFIN.forEach(function(input) {
+            printData(input.text,parseInt(input.id),false)
+        });
+    }
+}
+
+function saveDATA(){
+    localStorage.setItem(TODODATA, JSON.stringify(toDoArr));
+    localStorage.setItem(FINDATA, JSON.stringify(finArr));
+}
+
+function setLength(){
+    lenTodo.innerText = toDoArr.length
+    lenFin.innerText = finArr.length
+}
+
+function deleteData(arr,target){
+    // filter target from arr
+    // target.id가 string으로 가져와짐 - 그래서 처음에 값을 비교를 못했음
+    const result = arr.filter(element => element.id!==target.id)
+    return result
+}
+
+function deleteHandler(event){
+    // event.target의 class를 보고 todo 인지 finished인지 판별 가능
+    const target = event.target.parentNode.parentNode
+    if(target.className === "todo"){
+        // delete from todoArr
+        toDoArr = toDoArr.filter(element => element.id !== target.id)
+        // remove from parent
+        toDoList.removeChild(target);
+    }else{
```

Comment:
```suggestion
    }else if(target.className === 'fin') {
```

물론 이번 과제에서 `todo` 상태의 반대는 `fin`임이 자명하지만, `fin` 상태를 명시함으로서 가독성을 더 높일수 있을 것 같습니다!

Reply:
이런 디테일 한 스푼이 가독성을 높이는데, 너무 아쉬운것 같습니다ㅠ